oc-chef-pedant/lib/pedant/platform.rb

```diff
@@ -75,7 +75,9 @@ def map_to_default_orgname?(path_fragment)
     def api_url(path_fragment = '/', org=test_org)
       path_prefix = (map_to_default_orgname?(path_fragment) ? '' : "/organizations/#{org.name}")
       slash = path_fragment.start_with?('/') ? '' : '/'
-      "#{server}#{path_prefix}#{slash}#{path_fragment}"
+z =   "#{server}#{path_prefix}#{slash}#{path_fragment}"
```

Comment:
Is this intended to be temp/debugging code?

Reply:
Yes - sorry, this PR is still being worked on.  Any cruft, instrumentation, etc. will be cleaned-up before merge.